service/routes.py

```diff
@@ -66,6 +66,18 @@ def create_recommendations():
         jsonify(message), status.HTTP_201_CREATED
     )
 
+@app.route("/recommendations/<int:product_id1>/related-products/<int:product_id2>", methods=["DELETE"])
+def delete_recommendations(product_id1, product_id2):
+    """
+    Delete a relationship
+    This endpoint will delete a relationship based the product ids specified in the path
+    """
+    app.logger.info("Request to delete relationship between product ids: %s %s", product_id1, product_id2)
+    recommendation = Recommendation.find(product_id1, product_id2)
+    if recommendation:
```

Comment:
What if the recommendation is not found? Should we return errors here and also add a test?

Reply:
Some reference here about not found resources. https://stackoverflow.com/questions/17884469/what-is-the-http-response-code-for-failed-http-delete-operation

Reply:
Done. I don't know why if I don't add method GET in route will cause 405 method not allowed. In professor's lab-flask-tdd it just return 404. I did this in an ugly way. If you have any idea please notice me.

Reply:
I am going to guess it's because the uri path is the same as READ point somehow and I think it's fine to add GET there as long as all tests still pass.